CONTRIBUTING.md

```diff
@@ -95,9 +95,10 @@ changed and why. Follow these guidelines when writing one:
 1. The first line should be 50 characters or less and contain a short
    description of the change. All words in the description should be in
    lowercase with the exception of the ones that refer to code (e.g.
-   function/variable names). The description should be prefixed with the name
-   of the changed subsystem and start with a present-tense or imperative verb
-   (e.g. "net: add localAddress and localPort to Socket").
+   function/variable names), proper nouns, and acronyms. The description should
+   be prefixed with the name of the changed subsystem and start with a
+   present-tense or imperative verb (e.g. "net: add localAddress and localPort
```

Comment:
I'm fine with this landing as-is. I want to add: I know I'm the person who said `present-tense` may be preferable to `imperative verb`. I'm starting to have second thoughts. `fixing` would be present tense but is not OK. An imperative verb will always be OK (I think). So if you'd prefer to just use `imperative` and drop `present-tense`, I'd be OK with that.

So, yeah, either way, LGTM.

Nit: I prefer we write out `for example` rather than rely on `e.g.`.  Again, LGTM as is, and LGTM with that change. Either way.